### PR TITLE
Userdashboard retaining shelters animals and white font

### PR DESCRIPTION
**Pull request by tmitchellisaac**
Started adding heroku stubs to lots of tests. There are still some tests not passing that are not explicitly from webmock stubs/api calls, just an fyi

### CODE REVIEW

**Comment by yainporter** [comment]
I noticed some of the changes still have the stub request routes to localhost:5000, is there a reason why you add that one along with the heroku stubs?

File: app/controllers/animals_controller.rb
Comment (s2an) [inline app/controllers/animals_controller.rb:1]: Nice use of microservice / SRP!

File: spec/features/animals/delete_animal_spec.rb
Comment (s2an) [inline spec/features/animals/delete_animal_spec.rb:1]: Ugh. I think from now on I will always put all stub requests into a before each block whenever possible! You for sure put in a lot of time getting them all done. I wish there was a better way?!

File: spec/features/users/new_spec.rb
Comment (s2an) [inline spec/features/users/new_spec.rb:1]: It looks like you started doing the before each :)

File: app/services/animal_service.rb
Comment (s2an) [inline app/services/animal_service.rb:1]: If there is time, hopefully we can make these more restful.

**Comment by s2an** [comment]
Ready to merge once I get the okay (since I was gone a while). Looks like it was tedious!

Yain, I believe they are both there that way it would still work if there is the need to switch to localhost to test it. We ran into a problem with testing and needing to switch to Heroku while it was deployed.

**Pull request merged**